webapp/channels/src/actions/invite_actions.ts

```diff
@@ -34,9 +35,9 @@ export function sendMembersInvites(teamId: string, users: UserProfile[], emails:
         for (const user of users) {
             const member = getTeamMember(state, teamId, user.id);
             if (isGuest(user.roles)) {
-                notSent.push({user, reason: localizeMessage('invite.members.user-is-guest', 'Contact your admin to make this guest a full member.')});
+                notSent.push({user, reason: defineMessage({id: 'invite.members.user-is-guest', defaultMessage: 'Contact your admin to make this guest a full member.'})});
```

Comment:
I had to partly revert that because we still had code that set `reason` to be an error string from the server in some cases. Unfortunately, I didn't catch that earlier because I assumed that the type check would fail when I made it not possible for it to be a string, but because `response.error` is an `any`, it wasn't reported